File: app/static/js/ats_check.js
document.getElementById('atsCheckForm').addEventListener('submit', async (event) => {
    event.preventDefault();

    let resumeFile = document.getElementById('resumeFile').files[0];
    let formData = new FormData();
    formData.append('file', resumeFile);

    let response = await fetch('/ats_check', {
        method: 'POST',
        body: formData
    });

    if (response.ok) {
        let result = await response.json();
        if (result.message === 'ATS_CHECK_SUCCESS') {
            document.getElementById('atsResult').innerText = 'Your resume has passed the ATS check.';
        } else {
            document.getElementById('atsResult').innerText = 'Your resume did not pass the ATS check. Please review and make necessary changes.';
        }
    } else {
        console.error('Error in ATS check:', response.statusText);
    }
});

File: app/routers/ats_check.py
```python
from fastapi import APIRouter, HTTPException, UploadFile, File
from app.services.ats_check import check_ats

router = APIRouter()

@router.post("/ats-check")
async def ats_check(resume: UploadFile = File(...)):
    try:
        result = await check_ats(resume)
        return {"message": "ATS_CHECK_SUCCESS", "result": result}
    except Exception as e:
        raise HTTPException(status_code=400, detail=str(e))
```